### DISCUSSION TITLE
Alternative Driver

### DISCUSSION THREAD

**Discussion by AxelJust** [LydByDissing/stroem#34] [General]
Hey,
I have the Time and motivation this week to build a couple of them and I'd Like to Try some other speakers as well to do some mesurements.
Do You have any recomendations I shoud try?
greatings
Axel

**Comment by TooDissing** [comment]
> Hey,
> I have the Time and motivation this week to build a couple of them and I'd Like to Try some other speakers as well to do some mesurements.
> Do You have any recomendations I shoud try?
> greatings
> Axel

Sounds great Axel! Please do not hesitate to reach out in case there's something that doesn't make sense :-)

I do not have any driver alternatives on hand, as we have designed the cabinet and the tuning/voicing with these drivers. If you try out additional drivers, then you pretty much starts from scratch. Not saying that using different drivers are impossible, but more work are required ☕️☕️☕️
